pyfr/plugins/fluidforce.py

```diff
@@ -145,6 +173,12 @@ def __call__(self, intg):
 
             # Do the quadrature
             f[:ndims] += np.einsum('i...,ij,jik', qwts, p, norms)
+            if self._mcomp:
+                rfpts_c_norms = self._rfpts_c_norms[etype, fidx]
+
+                # Pressure force moments
+                mop = 'i...,ij,ji' if self.ndims == 2 else 'i...,ij,jik'
```

Comment:
Can we unify these through broadcasting?

Reply:
I used `np.atleast_3d` to unify, let me know if that's OK